### DISCUSSION TITLE
Satu Sehat  Id_procedure & Id_Composition yang non budled

### DISCUSSION THREAD

**Discussion by jokoblitar** [basoro/Khanza-Lite#153] [General]
Kami butuh Satu Sehat  Id_procedure & Id_Composition yang non budled;  karena kalau yang budled sepertinya tidak tercatat responya;  

Kalau ada yang non bundled. Komplet Deh.  Thank.s drg. basoro

**Comment by basoro** [comment]
Kalo yg bundled dan tidak tersimpan responnya dikarenakan ada kolom yang kurang pada tabel mlite_satu_sehat_response. 
Silahkan lihat pada commit https://github.com/basoro/mlite/commit/a8381c3d3820eccaffa663c0a5a893f22254cf06 untuk lebih detailnya.

Salam
